recipes/seadex-essentials/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile, CMake
+from conan.tools.build import cross_building
+import os
+
+
+# legacy validation with Conan 1.x
+class TestPackageV1Conan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+
+    default_options = {
+        "spdlog/1.11.0:header_only": True
+    }
```

Comment:
I think you might be able to remove this after making the changes I previously mentioned regarding setting `header_only` in the `configure` method if it is necessary. You'll want to remove this.

```suggestion
```